src/nearby-stop-places/NearbyStopPlacesScreenComponent.tsx

```diff
@@ -58,6 +58,8 @@ export const NearbyStopPlacesScreenComponent = ({
 
   const screenHasFocus = useIsFocused();
 
+  const onSelectStopPlaceRef = useRef(onSelectStopPlace);
```

Comment:
Should rather use `useCallback` on the input functions instead of `useRef` here? I know `useRef` feels safer, but I think we should rather do it the more correct React way.

Reply:
refactored to use `useCallback`